<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qdate</class>
 <widget class="QWidget" name="Qdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="inputlineEdit1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>112</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputname1">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>42</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入项目</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="inputname2">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入项目</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputlineEdit2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelcm">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单位均为厘米</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
